internal/pkg/aws/sessions/errors.go

```diff
@@ -24,3 +24,22 @@ func (e *errMissingRegion) RecommendActions() string { // implements new actionR
 - Alternatively, you can run %s to set the environment variable.
 More information: https://aws.github.io/copilot-cli/docs/credentials/`, color.HighlightCode("export AWS_REGION=<application region>"))
 }
+
+type errCredProviderTimeout struct{}
+
+// Implements error interface.
+func (e *errCredProviderTimeout) Error() string {
+	return "retrieving credentials timeout"
+}
+
+// RecommendActions returns recommended actions to be taken after the error.
+// Implements main.actionRecommender interface.
+func (e *errCredProviderTimeout) RecommendActions() string {
+	return fmt.Sprintf(`It looks like your %s chain is missing or misconfigured:
+https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html#specifying-credentials
+- We recommend including your credentials in the shared credentials file.
+- Alternatively, you can also set credentials throguh 
+	* Environment Variables
+	* EC2 Instance Metadata (credentials only)
+More information: https://aws.github.io/copilot-cli/docs/credentials/`, color.HighlightCode("default AWS credential"))
```

Comment:
If the err occurs in the `FromProfile` method then it's not default cred chain.

Perhaps we can have a field in the type like this:
```go
//errCredProviderTimeout indicates a timeout occurred while fetching credentials for a session.
// The zero value errCredProviderTimeout{} represents the default AWS credential chain.
type errCredProviderTimeout struct{
    profile string // Optional. If the credentials were created from a profile. 
}
```